bpf-gpl/nat.h

```diff
@@ -188,7 +211,10 @@ static CALI_BPF_INLINE struct calico_nat_dest* calico_v4_nat_lookup2(__be32 ip_s
 		}
 		CALI_DEBUG("NAT: nodeport hit\n");
 	}
-
+	if (nat_lv1_val->id == 0xffffffff && nat_lv1_val->count == 0) {
+		*drop = 1;
```

Comment:
And preferably, `0xffffffff` defined as something like `REJECTED_SRC_IP_ID` so that it is easier to read the code and match it with a similar constant in golang.

Since we use cgo already, perhaps we should move such constants in some header file that we can include in go.